sdcm/nemesis.py

```diff
@@ -1525,16 +1525,18 @@ def disrupt_mgmt_backup(self):
         self._mgmt_backup(backup_specific_tables=False)
 
     def _mgmt_backup(self, backup_specific_tables):
-        # TODO: When cloud backup is supported - add this to the below 'if' statement:
-        #  and not self.cluster.params.get('use_cloud_manager')
-        if not self.cluster.params.get('use_mgmt'):
+        if not (self.cluster.params.get('use_mgmt') or
+                self.cluster.params.get('use_cloud_manager')):
             raise UnsupportedNemesis('Scylla-manager configuration is not defined!')
-        if not self.cluster.params.get('backup_bucket_location'):
-            raise UnsupportedNemesis('backup bucket location configuration is not defined!')
 
         mgr_cluster = self.cluster.get_cluster_manager()
         cluster_backend = self.cluster.params.get('cluster_backend')
-        backup_bucket_location = self.cluster.params.get('backup_bucket_location')
+        if self.cluster.params.get('use_mgmt', default=None):
```

Comment:
there no more `default` for `params.get`

Reply:
Yes, sorry. Forgot to remove this one during the rebase